Documentation/ceph-disaster-recovery.md

```diff
@@ -336,3 +336,17 @@ Assuming `dataHostPathData` is `/var/lib/rook`, and the `CephCluster` trying to
 1. Bring the Rook Ceph operator back online by running `kubectl -n rook-ceph edit deploy/rook-ceph-operator` and set `replicas` to `1`.
 1. Watch the operator logs with `kubectl -n rook-ceph logs -f rook-ceph-operator-xxxxxxx`, and wait until the orchestration has settled.
 1. **STATE**: Now the new cluster should be up and running with authentication enabled. `ceph -s` output should not change much comparing to previous steps.
+
+## Adopting into a new Kubernetes cluster with PVCs
+
+It is possible to migrate/restore an rook/ceph cluster from an exising Kubernetes cluster to a new one without restoriting to SSH access or ceph tooling. This allows doing the migration using standard kubernetes resources only. This guide assumes you have a CephCluster that uses PVCs to persist mon and osd data.
+1. Stop rook in the cluster cluster by scaling the operator deployment `rook-ceph-operator` down to zero and deleting the other deployments: `rook-ceph-mgr-a`, `rook-ceph-mon-a`, ..., `rook-ceph-osd-0`, ...
```

Comment:
```suggestion
1. Stop rook in the old cluster by scaling the operator deployment `rook-ceph-operator` down to zero and deleting the other deployments: `rook-ceph-mgr-a`, `rook-ceph-mon-a`, ..., `rook-ceph-osd-0`, ...
```

Reply:
I actually meant the new cluster. restructured the description to make it clearer.